pkg/netceptor/conn_test.go

```diff
@@ -232,3 +233,14 @@ func TestSetWriteDeadline(t *testing.T) {
 		}
 	})
 }
+
+func TestListen(t *testing.T) {
+	host_names := []string{"127.0.0.1", "1.2.3.4:5000", "::1", "[::1]:2000", "1:2:3:4:5:6:7:8", "[1:2:3:4:5:6:7:8]:9000"}
```

Comment:
```suggestion
	hosts := []string{"127.0.0.1", "1.2.3.4:5000", "::1", "[::1]:2000", "1:2:3:4:5:6:7:8", "[1:2:3:4:5:6:7:8]:9000"}
```
Style change but "hosts" is likely to be cleaner here